base/linalg/generic.jl

```diff
@@ -986,7 +986,7 @@ function ishermitian(A::AbstractMatrix)
         return false
     end
     for i = indsn, j = i:last(indsn)
-        if A[i,j] != adjoint(A[j,i])
+        if A[i,j] != Adjoint(A[j,i])
```

Comment:
Why are you favoring the uppercase constructors in these cases?

Reply:
The simple answer is that this pull request presently represents an intermediate state which will change substantially on next push. Specifically, most uppercase calls should then disappear :).